### DISCUSSION TITLE
v24.10.08

### DISCUSSION THREAD

**Discussion by davidrapan** [davidrapan/ha-solarman#201] [Announcements]
## What's Changed
* [refactor: Processing all modbus responses in bulk](https://github.com/davidrapan/ha-solarman/commit/949dec5bac96ffd9714ed8a7f702e809670243d3)

**Full Changelog**: https://github.com/davidrapan/ha-solarman/compare/v24.10.04...v24.10.08

<hr /><em>This discussion was created from the release <a href='https://github.com/davidrapan/ha-solarman/releases/tag/v24.10.08'>v24.10.08</a>.</em>

**Comment by PV-Joe** [comment]
ha-solarman v24.10.08 ran stable.

Updated to v24.10.10 right now and everything seems to work fine 👍
